### DISCUSSION TITLE
old devices

### DISCUSSION THREAD

**Discussion by ha4bf** [N0BOY/FT8CN#19] [Ideas]
Hi. 
Would it be possible to make a build that could also be installed on an Android 5.1.1 tablet?
73, ha4bf

**Comment by TF1979** [comment]
I would like the same, since my tablets are oldish.

**Comment by ra3trm** [comment]
对不起！
将CAT YAESU FT-747gx添加到FT8CN程序中。
为什么在WJTDX中有一个YAESU FT-747GX，但在FT8CN中没有？
不是每个人都有最新的收发器！

谢谢！Sorry!
Add the CAT YAESU FT-747GX to the FT8CN program.
Why is there a YAESU FT-747GX in WJTDX, but not in FT8CN ?
Not everyone has the latest transceivers!

THANKS!
